pypet2bids/pypet2bids/ecat2nii.py

```diff
@@ -132,7 +132,11 @@ def ecat2nii(ecat_main_header=None,
             prompts.append(0)
             randoms.append(0)
 
-    final_image = img_temp * main_header['ECAT_CALIBRATION_FACTOR']
+    ecat_cal_units = main_header['CALIBRATION_UNITS']  # Header field designating whether data has already been calibrated
+    if ecat_cal_units==0:                              # Calibrate if it hasn't been already
+        final_image = img_temp * main_header['ECAT_CALIBRATION_FACTOR']
+    elif ecat_cal_units==1:                            # And don't calibrate if it has
+        final_image = img_temp
```

Comment:
```suggestion
final_image = img_temp * main_header['ECAT_CALIBRATION_FACTOR']  
```

Reply:
So I updated `scl_slope` in your branch, I think this fixes the need for the above changes you made. If you would like @noahg-neuroimage you can try it out yourself. My testing locally shows that it seems to be the way to go.

Apologies for all the headache.

Reply:
is img_temp already dcaled to 16 bits here? (in the matlab code it isn't)

Reply:
I agree with @bendhouseart commit https://github.com/openneuropet/PET2BIDS/commit/f1127f579fdf4389a5c3dcb9ed4d2dfcc23a0016 let's not touch the slope

Reply:
> is img_temp already scaled to 16 bits here? (in the matlab code it isn't)

Let me get back to you on that ;), I'm going to make some nifti's with this branch as is and tag them with the commit hash first.